server/graphql/v2/object/TransactionGroup.ts

```diff
@@ -0,0 +1,54 @@
+import { GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { GraphQLDateTime } from 'graphql-scalars';
+
+import { sequelize, Transaction } from '../../../models';
+import { GraphQLAccount } from '../interface/Account';
+import { GraphQLTransaction } from '../interface/Transaction';
+import { GraphQLAmount } from '../object/Amount';
+import { getTransactionKindPriorityCase } from '../query/collection/TransactionsCollectionQuery';
+
+export const GraphQLTransactionGroup = new GraphQLObjectType({
+  name: 'TransactionGroup',
+  description: 'Transaction group',
+  fields: () => {
+    return {
+      id: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+      amount: {
+        type: new GraphQLNonNull(GraphQLAmount),
+      },
```

Comment:
It's unclear what "amount" would be for a transaction group. Since it contains DEBITs and CREDITs, the sum of all transactions in a group would normally be 0. If we intend to filter in the frontend, I'd suggest renaming this to `totalAmount` to make it clearer that it's a sum.

Since this is a beta query, I'm only thinking out loud in these comments - this is not a requested change.

Reply:
Since we're only fetching the transactions in a group on a specific account (and not all the transactions in the group) the amount will not be 0. This is at least the current implementation, with the required `account` argument in the queries.

`totalAmount` makes sense, or possibly `netAmount`?

Reply:
`netAmount` could be confusing as we use this term in many contexts to signify "amount without fees / taxes".